.env_config_sample

```diff
@@ -1,2 +1,2 @@
-export DATABASE_URI='postgres://USER:PASSWORD@HOST:PORT/NAME'
-export SECRET_KEY='AuthorsHaven'
+export DATABASE_URI='postgres://postgres:password@localhost:5432/authors_haven'
```

Comment:
@ezrogha please revert the these changes

Reply:
Thanks for the feedback and assistance @ja-odur